package app.test.networkapp.data.models.exception

class OrganizationNotFoundException : Exception("Организация не найдена")